src/Components/blinds.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import Open from '../Images/blinds-open.svg';
+import Closed from '../Images/blinds-closed.svg';
+import './blinds.css';
+
+const Blinds = ({ state }) => {
+    const [isOpen, setIsOpen] = useState(state === "open")
+
+    const handleClick = () => {
+        setIsOpen(!isOpen);
+    }
+
+    return (
+        <div className="blinds">
+            <div className="blinds__icon">
+                <img src={isOpen ? Open : Closed} alt={isOpen ? "zaluzie otevreny" : "zaluzie zatazeny"}/>
+            </div>
+            <div className="blinds__name">
+                Žaluzie
+            </div>
+            <div className="blinds__controls">
+                <button className={isOpen ? "button button--active" : "button"} disabled={isOpen} onClick={handleClick}>Otevřeno</button>
```

Comment:
tohle už je trochu pokročilejší, ale když máš tlačítko, které zapíná, tak ta funkce by neměla "togglovat" to světlo, ale jen zapínat. tj správně by mělo být
`<button className={isOpen ? "button button--active" : "button"} disabled={isOpen} onClick={() => handleClick(false)}>Otevřeno</button>` (a true u dalšího řádku)
A to proto, že kdyby někdo udělal chybu nebo záměrně odstranil to disabled=xxx, tak najednou máš nekonzistenci. Proto je dobré držet se principu "dělá to to, jak se to jmenuje".
Ale to už je trochu pokročilý mindset, jinak to samozřejmě funguje :)